ssas/testutils.go

```diff
@@ -136,3 +135,32 @@ func CreateTestXData(t *testing.T, db *gorm.DB) (creds Credentials, group Group)
 
 	return
 }
+
+func CreateTestXDataV2(t *testing.T, db *gorm.DB) (creds Credentials, group Group) {
+	groupID := RandomHexID()[0:4]
+
+	group = Group{GroupID: groupID, XData: "fake x_data"}
```

Comment:
Same as below: let's go for some valid JSON
```suggestion
	group = Group{GroupID: groupID, XData: `{"key":"value"}`